crates/sui-graphql-rpc/src/main.rs

```diff
@@ -112,3 +133,7 @@ fn service_config(path: Option<PathBuf>) -> ServiceConfig {
     let contents = fs::read_to_string(path).expect("Reading configuration");
     ServiceConfig::read(&contents).expect("Deserializing configuration")
 }
+
+fn version(version: &'static str) -> Version {
```

Comment:
As a simplification, I would personally just make `Version` a tuple-struct: `pub struct Version(&'static str)`. Also, I think you should be able to make `static VERSION` have type `Version` to simplify another step.

Reply:
Simplified it, thanks.